packages/components/src/inputs/src/PasswordField.module.css

```diff
@@ -0,0 +1,33 @@
+.hop-PasswordField  {
+    --hop-PasswordField-gap: var(--hop-space-stack-xs);
+    --hop-PasswordField-default-inline-size: 20rem;
+
+    display: flex;
+    flex-direction: column;
+    gap: var(--hop-PasswordField-gap);
+    inline-size: var(--hop-PasswordField-default-inline-size);
+}
+
+.hop-PasswordField--fluid  {
+    inline-size: 100%;
```

Comment:
oh nevermind, there is one. This should be a component variable and we should be setting --inline-size here instead.
The inline-size property should be set on .hop-PasswordField.
Basically, props are set inside selectors that have --something on the end. rarely does it have properties being set.